unit-cloud/aos-unit-messages.schema.json

```diff
@@ -0,0 +1,2657 @@
+                        "$ref": "#/$defs/AosDesiredComponentInfo"
+                    },
+                    "title": "Components",
+                    "type": "array"
+                },
+                "layers": {
+                    "default": null,
+                    "description": "list of the desired layers",
+                    "items": {
+                        "$ref": "#/$defs/AosDesiredLayerInfo"
+                    },
+                    "title": "Layers",
+                    "type": "array"
+                },
+                "services": {
+                    "default": null,
+                    "description": "List of the desired services. If absent or null - do nothing.",
+                    "items": {
+                        "$ref": "#/$defs/AosDesiredServiceInfo"
+                    },
+                    "title": "Services",
+                    "type": "array"
+                },
+                "instances": {
+                    "default": null,
+                    "description": "List of the desired services instances. If absent or null - do nothing.",
+                    "items": {
+                        "$ref": "#/$defs/AosDesiredInstanceInfo"
+                    },
+                    "title": "Instances",
+                    "type": "array"
+                },
+                "fotaSchedule": {
+                    "allOf": [
+                        {
+                            "$ref": "#/$defs/AosScheduleRule"
+                        }
+                    ],
+                    "default": null,
+                    "description": "Points to rules when FOTA can be applied."
+                },
+                "sotaSchedule": {
+                    "allOf": [
+                        {
+                            "$ref": "#/$defs/AosScheduleRule"
+                        }
+                    ],
+                    "default": null,
+                    "description": "Points to rules when SOTA can be applied."
+                },
+                "certificates": {
+                    "default": null,
+                    "description": "The list of the used certificates",
+                    "items": {
+                        "$ref": "#/$defs/AosCertificateInfo"
+                    },
+                    "title": "Certificates",
+                    "type": "array"
+                },
+                "certificateChains": {
+                    "default": null,
+                    "description": "certificate chains info for checking signs",
+                    "items": {
+                        "$ref": "#/$defs/AosCertificateChainInfo"
+                    },
+                    "title": "Certificatechains",
+                    "type": "array"
+                }
+            },
+            "required": [
+                "messageType"
+            ],
+            "title": "AosDesiredStatus",
+            "type": "object"
+        },
+        "AosEnvVar": {
```

Comment:
Could you group messages by their usage? For example: all env related messages in one place?